application/src/main/java/org/togetherjava/tjbot/commands/help/BookmarksHelper.java

```diff
@@ -0,0 +1,109 @@
+package org.togetherjava.tjbot.commands.help;
+
+import org.togetherjava.tjbot.db.Database;
+import org.togetherjava.tjbot.db.generated.tables.records.BookmarksRecord;
+import org.togetherjava.tjbot.db.generated.tables.records.HelpThreadsRecord;
+
+import javax.annotation.Nullable;
+
+import java.time.Instant;
+import java.time.temporal.ChronoUnit;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+
+import static org.togetherjava.tjbot.db.generated.Tables.BOOKMARKS;
+import static org.togetherjava.tjbot.db.generated.Tables.HELP_THREADS;
+
+/**
+ * This is a utility class for interacting with the database
+ */
+public class BookmarksHelper {
+
+    private BookmarksHelper() {
+        throw new IllegalStateException("Utility class");
+    }
+
+    protected static Optional<HelpThreadsRecord> getHelpThread(Database database, long channelId) {
```

Comment:
why are all methods in this class protected? probably meant to use package-private